anyblock-uniswap-vwap/README.md

```diff
@@ -0,0 +1,51 @@
+# Chainlink External Adapter using Anyblock Analytics service to calculate VWAP for any Uniswap asset
+
+You must provide the api key using the env variable `API_KEY`!
+
+## Input Params
+
+- `address`:
+- `debug`: Switch to show `raw` trade values (optional, default: false)
+- `roundDay` Switch to round the start and end to midnight UTC (optional, default: false)
+- `start`: Epoch timestamp in seconds (optional, default: \$now - 24h)
+- `end`: Epoch timestamp in seconds (optional, default: \$now)
+
+Uniswap Offshift (XTF) example:
+
+```json
+{
+  "id": "1",
+  "data": {
+    "address": "0x2B9e92A5B6e69Db9fEdC47a4C656C9395e8a26d2"
+  }
+}
+```
+
+## Output Format
+
+```json
+{
+  "jobRunID": "1",
+  "data": {
+    "result": {
+      "volume": 3.447481508300616e22,
+      "vwap": 299.2532855156056
+    },
+    "raw": [
+      {
+        "timestamp": 1600304409,
+        "reserve0": 1.3224814613263435e23,
+        "reserve1": 406170744097414250000
+      },
+      {
+        // more raw values
+      }
+    ]
+  },
+  "result": {
```

Comment:
We've typically only put one value in the result key. This is so that we can have different adapter implementations (for different providers) but give the same job spec to multiple node operators. In this case, the `copyPath` would need to be `["result", "vwap"]` for the vwap value instead of just result. Is there a hard reason for this or can we just return the vwap value in result?

Reply:
@jondoe1337 how do you plan tu use this adapter?

Reply:
Uhm well, for now it might just be used in the XFT Adapter #87
However I'm pretty sure that there might be other price feeds wanted in future!?

Reply:
If this adapter is going to be used with other price feed adapters, then only the value within vwap should go to the result key.

Reply:
I've updated the adapter so it now only outputs the vwap value as result.